### PR TITLE
Improve examples in lang rules (5b7ae0 / bf051a / b5c3f8)

### PR DESCRIPTION
**Pull request by Jym77**
Add some missing inapplicable examples.
Reorder inapplicable examples to have a logical order following Applicability.

Closes issue(s):
- #1094 
- #1097 

Need for Final Call:
This will require a 1 week Final Call (adding several examples)

---

### **When creating PR:**

- Make sure you requesting to **pull a branch** (right side) to the `develop` branch (left side).

### **After creating PR:**

- Add yourself (and co-authors) as "Assignees" for PR.
- Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- Close the issue that the PR resolves (and make sure the issue is referenced in the top of this comment)
- Optionally request feedback from anyone in particular by assigning them as "Reviewers".

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/html-page-lang-b5c3f8.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ This rule applies to any [document element](https://dom.spec.whatwg.org/#documen
 
 ## Expectation
 
-Each test target has a `lang` attribute that is neither empty ("") nor only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace).
+Each test target has a `lang` attribute that is neither empty (`""`) nor only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace).
 
 ## Assumptions
 
-This rule assumes that the presence of a `lang` attribute is being used to comply to WCAG. This rule doesn't test if the attribute is needed to comply to WCAG.
+This rule assumes that the presence of a `lang` attribute is being used to comply to [WCAG][]. This rule doesn't test if the attribute is needed to comply to [WCAG][].
```
Comment (WilcoFiers) [inline _rules/html-page-lang-b5c3f8.md]: This seems unnecessary. Nobody reading this document is going to be confused about what WCAG is or where to find it.

File: _rules/html-page-lang-b5c3f8.md [hunk 2]
```diff
@@ -29,11 +29,11 @@ This rule applies to any [document element](https://dom.spec.whatwg.org/#documen
 
 ## Expectation
 
-Each test target has a `lang` attribute that is neither empty ("") nor only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace).
+Each test target has a `lang` attribute that is neither empty (`""`) nor only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace).
```
Comment (WilcoFiers) [inline _rules/html-page-lang-b5c3f8.md]: Why are you changing this?

File: _rules/html-page-lang-valid-bf051a.md [hunk 3]
```diff
@@ -66,26 +66,18 @@ This `html` element has a `lang` attribute whose value is a [valid][valid langua
 This `html` element has a `lang` attribute whose value is not a [valid language subtag][].
 
 ```html
-<html lang="xyz"></html>
+<html lang="enn-US"></html>
```
Comment (WilcoFiers) [inline _rules/html-page-lang-valid-bf051a.md]: "enn" is a valid language tag. I don't think this example needs to be changed.

File: _rules/html-page-lang-valid-bf051a.md [hunk 4]
```diff
@@ -66,26 +66,18 @@ This `html` element has a `lang` attribute whose value is a [valid][valid langua
...
-<html lang="en-US-GB"></html>
-```
-
-#### Failed Example 3
-
 This `html` element has a `lang` attribute whose value is not a [valid language subtag][].
 
 ```html
-<html lang="123"></html>
+<html lang="#1"></html>
```
Comment (WilcoFiers) [inline _rules/html-page-lang-valid-bf051a.md]: Why are you changing this?

File: _rules/html-page-lang-xml-lang-match-5b7ae0.md [hunk 5]
```diff
@@ -113,44 +121,55 @@ This rule does not apply to `svg` elements, even inside an `html` element.
...
 ```
 
 #### Inapplicable Example 6
 
-This rule does not apply to elements without a `xml:lang` attribute.
+This rule does not apply to `html` elements whose `lang` attribute is not a [valid language subtag][].
 
 ```html
-<html lang="en"></html>
+<html lang="enn" xml:lang="en"></html>
```
Comment (WilcoFiers) [inline _rules/html-page-lang-xml-lang-match-5b7ae0.md]: "enn" is a valid language subtag. Additionally, this example fails SC 3.1.1. We shouldn't put examples like that in inapplicable.

File: _rules/html-page-lang-b5c3f8.md (hunk 2 above)
Comment (ShadowBB) [reply _rules/html-page-lang-b5c3f8.md]: I think, technically this is better. 

("") 
could mean 
attribute="value" lang= attribute "value"

While 
(`""`) 
explicitly means
attribute="value" lang="" attribute "value"

But it is a really small difference.

File: _rules/html-page-lang-valid-bf051a.md (hunk 3 above)
Comment (Jym77) [reply _rules/html-page-lang-valid-bf051a.md]: Changing according to https://github.com/act-rules/act-rules.github.io/issues/1097#issuecomment-573747130
(agreed that "enn" is bad choice).

File: _rules/html-page-lang-valid-bf051a.md (hunk 4 above)
Comment (Jym77) [reply _rules/html-page-lang-valid-bf051a.md]: Changing according to https://github.com/act-rules/act-rules.github.io/issues/1097#issuecomment-573747130

File: _rules/html-page-lang-valid-bf051a.md (hunk 3 above)
Comment (EmmaJP) [reply _rules/html-page-lang-valid-bf051a.md]: Can someone suggest a better alternative

File: _rules/html-page-lang-xml-lang-match-5b7ae0.md (hunk 5 above)
Comment (EmmaJP) [reply _rules/html-page-lang-xml-lang-match-5b7ae0.md]: Can someone suggest better alternative

File: _rules/html-page-lang-valid-bf051a.md (hunk 4 above)
Comment (EmmaJP) [reply _rules/html-page-lang-valid-bf051a.md]: Example of an invalid value - numbers and non-alphanumeric characters are both invalid but don't necessarily need to be separate examples

File: _rules/html-page-lang-valid-bf051a.md (hunk 3 above)
Comment (Jym77) [reply _rules/html-page-lang-valid-bf051a.md]: Replaced with `em` which as far as I can tell is not a valid primary language tag.

File: _rules/html-page-lang-xml-lang-match-5b7ae0.md [hunk 6]
```diff
@@ -112,44 +120,55 @@ This rule does not apply to `svg` elements, even inside an `html` element.
...
 ```
 
 #### Inapplicable Example 4
 
-This rule does not apply to `html` elements whose `lang` attribute is not a [valid language subtag][], such as the empty string.
+This rule only applies to `html` in [top-level browsing context][]
 
 ```html
-<html lang="" xml:lang=""></html>
+<iframe srcdoc="<html lang='en' xml:lang='en'></html>" />
```
Comment (WilcoFiers) [inline _rules/html-page-lang-xml-lang-match-5b7ae0.md]: Can you give the page with the iframe an HTML element with a lang attribute?

File: _rules/html-page-lang-xml-lang-match-5b7ae0.md [hunk 7]
```diff
@@ -112,44 +120,55 @@ This rule does not apply to `svg` elements, even inside an `html` element.
...
 ```
 
 #### Inapplicable Example 5
 
-This rule does not apply to `html` elements whose `lang` attribute is not a [valid language subtag][], such as only ASCII whitespace.
+This rule only applies to documents with a [content type][] of `text/html`
 
-```html
-<html lang=" " xml:lang=""></html>
+```xhtml
```
Comment (WilcoFiers) [inline _rules/html-page-lang-xml-lang-match-5b7ae0.md]: I don't think our site knows how to handle this. @jkodu?

Comment (Jym77) [reply _rules/html-page-lang-xml-lang-match-5b7ae0.md]: It does.
https://github.com/act-rules/act-rules-web/pull/86

**Comment by Jym77** [comment]
Final Call ends on February 21st.

**Comment by Jym77** [comment]
Final Call has ended. Merging.

**Pull request merged**